packages/skills/carpark_detection/skill.yaml

```diff
@@ -0,0 +1,28 @@
+name: carpark_detection
+authors: Fetch.AI Limited
+version: 0.1.0
+license: Apache 2.0
+url: ""
+behaviours:
+  - behaviour:
+      class_name: ServiceRegistrationBehaviour
+      args: {}
+handlers:
+  - handler:
+      class_name: FIPAHandler
+      args: {}
+tasks: []
+shared_classes:
+  - shared_class:
+      class_name: Strategy
+      args:
+        data_price_fet: 2000
+       # db_is_rel_to_cwd: true
```

Comment:
Why are they commented out? @dishmop

Reply:
because the default behaviour is just to take dummy data from the directory supplied in the module.

Reply:
Fixes done ready for re-review.

@DavidMinarsch